src/App.js

```diff
@@ -14,8 +14,13 @@ import "./App.css";
 import Mockman from "mockman-js";
 import { ToastContainer } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
+import { useAuthContext } from "./Context/AuthContext/AuthContext";
+import RequireAuth from "./hooks/RequireAuth";
 
 function App() {
+
+  const { user } = useAuthContext()
+  console.log(user)
```

Comment:
remove console log

Reply:
sure i will remove it i still need it to check cart management and wishlist management hence kept it